frontend/src/components/about/BetaAlertDiv.tsx

```diff
@@ -0,0 +1,58 @@
+import { useTranslation } from 'react-i18next';
+import Alert from '@mui/material/Alert';
+import CloseIcon from '@mui/icons-material/Close';
+import { Button } from '@mui/material';
+
+type BetaAlertDivProps = {
+  setAlertVisible: (isVisible: boolean) => void;
+};
+
+const BetaAlertDiv: React.FC<BetaAlertDivProps> = ({ setAlertVisible }) => {
+  const { t } = useTranslation('about');
+
+  const handleClose = () => {
+    setAlertVisible(false);
+  };
+  return (
+    <Alert
+      id="closable-alert"
+      severity="info"
+      action={
+        <Button
+          aria-label="close"
+          sx={{ borderRadius: '8px', minWidth: '24px' }}
+          onClick={handleClose}
+        >
+          <CloseIcon sx={{ fontSize: '16px' }} />
+        </Button>
+      }
+      sx={{
+        fontSize: '18px',
+        width: '83.33%',
+        margin: '16px auto',
+        bgcolor: '#E5F6FD',
+        display: 'flex',
+        color: '#000',
+        fontWeight: '300',
+        border: '1px solid #666',
+        borderRadius: '8px',
+        alignItems: 'center',
+        '.MuiAlert-icon': {
+          fontSize: 30,
+        },
+      }}
+    >
+      {t('main.betaWarningFirstPart')}{' '}
+      <a
+        href="http://localhost:5173/about/contact"
```

Comment:
This is not the correct way to redirect. We don’t redirect to localhost because we need to be compatible with all environments where we deploy. Please take a look at other files how we do it.

Reply:
I'm embarrassed about local link, sorry for it. I changed it to href="/about/contact" as you use in repo that way.